platform/jvm/replay/src/main/kotlin/io/bitdrift/capture/replay/SessionReplayConfiguration.kt

```diff
@@ -9,10 +9,8 @@ package io.bitdrift.capture.replay
 
 /**
  * A configuration used to configure Bitdrift session replay feature.
- * @param captureIntervalMs The number of milliseconds between consecutive screen replay captures.
  * @param replayMapperConfiguration Map used to matching third party Android views to Bitdrift view types.
  */
 data class SessionReplayConfiguration @JvmOverloads constructor(
-    val captureIntervalMs: Long = 3000,
     val replayMapperConfiguration: ReplayMapperConfiguration? = null,
```

Comment:
🤔  apparently this is the only part of the configuration left, which makes the entire nested structure unnecessary complicated. not much to do now but leaving this message for posterity: we should simplify this